components/Player.js

```diff
@@ -0,0 +1,60 @@
+import propTypes from 'prop-types'
+import Card from './Card'
+
+/**
+ * Player list component
+ */
+export default function Player(props) {
+  return (
+    <div className="flex md:flex-col rounded border border-slate-300 my-2">
+      <div className=" flex w-full justify-between">
```

Comment:
I think `w-full` can be omitted here since it's being applied at the parent element `<li>`.

Reply:
Yes, it does. I think when you removed it you forgot to put a space between flex and justify-between